### ISSUE TITLE
The link with the URL is not accessible with the keyboard

### ISSUE THREAD

**Issue by TheOneRing** [owncloud/client#11772]
![320744148-611c5056-5f0d-49f8-9319-6f10aa3be9d8](https://github.com/user-attachments/assets/d52fef63-0863-4073-a87c-db95d0fcbd47)
